tripal/tripal
# Synonym field was broken on Drupal ≥ 10.2

# Bug Fix

### Closes #1817

### Tripal Version: 4.x

## Description
Since Drupal 10.2, the ChadoSynonymDefault field could not be added to a content type through the GUI, due to the subform introduced with this Drupal version.

One additional change:
`$storage_settings['storage_plugin_settings']['base_table'] = '';`
is present in the parent class, so I went ahead and removed it from all of the other fields too.

## Testing?
 1. Import genomic type collection
 2. Edit fields on "gene" and remove the existing synonym field
 3. Re-add it through the GUI. There should be no nasty errors now.
 4. Create a gene with a synonym and save it.
